src/form/actionCreators.js

```diff
@@ -31,9 +30,9 @@ export const clear = createAction(
 
 export const initialise = createAction(
   INITIALISE,
-  payload => omit(payload, ['form']),
+  ({ form, ...payload }) => payload,
   payload => ({
```

Comment:
Yeh, but it feel it could almost be a bit overkill. And it will stlll give linting errors. This is what I quickly whipped together. 
```javascript
export default (key, item) => {
  const {[key]: omit, ...rest} = item;
  return rest;
}
```